platform/11-tekton-chains.sh

```diff
@@ -11,29 +11,29 @@ GIT_ROOT=$(git rev-parse --show-toplevel)
 #     Possibles values: `intoto`. Any other value would simply use the default
 #     behaviour.
 
-: ${TKN_CHAINS_FORMAT:=""}
+: "${TKN_CHAINS_FORMAT:=""}"
 
 # Install Chains.
-kubectl apply --filename $GIT_ROOT/platform/vendor/tekton/chains/release.yaml
+kubectl apply --filename "$GIT_ROOT"/platform/vendor/tekton/chains/release.yaml
 
 kubectl patch \
       configmap chains-config \
       -n tekton-chains \
-      --patch-file $GIT_ROOT/platform/components/tekton/chains/patch_config_oci.yaml
+      --patch-file "$GIT_ROOT"/platform/components/tekton/chains/patch_config_oci.yaml
 
 # Patch chains to generate in-toto provenance.
 case "${TKN_CHAINS_FORMAT}" in
   intoto)
     kubectl patch \
       configmap chains-config \
       -n tekton-chains \
-      --patch-file $GIT_ROOT/platform/components/tekton/chains/patch_config_intoto.yaml
+      --patch-file "$GIT_ROOT"/platform/components/tekton/chains/patch_config_intoto.yaml
     ;;
   *)
     ;;
 esac
 
 # Install Cosign if needed.
 if ! cosign version; then
-  ./12-cosign-installer.sh
+  bash "$GIT_ROOT"/platform/12-cosign-installer.sh
```

Comment:
Doesn't seem like this will work if `12-cosign-installer.sh` has been removed.

Reply:
Ah shoot, missed that!